scripts/create_test

```diff
@@ -211,8 +323,38 @@ def create_test(testargs, compiler, machine_name, no_run, no_build, no_batch, te
                       compare=compare, generate=generate, namelists_only=namelists_only,
                       project=project, parallel_jobs=parallel_jobs,
                       xml_machine=xml_machine, xml_compiler=xml_compiler,
-                      xml_category=xml_category, xml_testlist=xml_testlist, walltime=walltime)
-    return 0 if impl.system_test() else CIME.utils.TESTS_FAILED_ERR_CODE
+                      xml_category=xml_category, xml_testlist=xml_testlist, walltime=walltime,
+                      proc_pool=proc_pool, use_existing=use_existing)
+
+    success = impl.system_test()
+
+    if single_submit:
+        job_cost_map = {}
+        largest_case = 0
+        for test in impl._tests:
+            test_dir = impl._get_test_dir(test)
```

Comment:
Why not just
case = Case(test_dir)
wallclock_time = case.get_value("JOB_WALLCLOCK_TIME",subgroup="case.test")